default.nix

```diff
@@ -0,0 +1,37 @@
+{ nixpkgs ? import ./nixpkgs.nix
+, pkgs ? import nixpkgs {}
+, version ? "dev"
+}:
+
+pkgs.buildGoModule rec {
+  name = "morph-unstable-${version}";
+  inherit version;
+
+  nativeBuildInputs = with pkgs; [ go-bindata ];
+
+  src = pkgs.nix-gitignore.gitignoreSource [] ./.;
+
+  buildFlagsArray = ''
+    -ldflags=
+    -X
+    main.version=${version}
+  '';
+
+  vendorSha256 = "08zzp0h4c4i5hk4whz06a3da7qjms6lr36596vxz0d8q0n7rspr9";
```

Comment:
If you store the dependencies in your Git repository then this can become set to null. See https://nixos.org/manual/nixpkgs/stable/#ssec-go-modules